core/src/components/toolbar/test/colors/toolbar.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('toolbar: colors', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/toolbar/test/colors`);
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot()).toMatchSnapshot(`toolbar-colors-${page.getSnapshotSettings()}.png`);
```

Comment:
Can we add a container around all of the rendered toolbars and capture a screenshot of that? The current images appear to capture a lot of whitespace.